tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Solution/ListSolutionsSteps.cs

```diff
@@ -84,6 +90,8 @@ public async Task ThenTheDetailsOfTheSolutionsReturnedAreAsFollows(Table table)
             var expectedSolutions = table.CreateSet<SolutionDetailsTable>();
```

Comment:
It might be worth adding a `ToList()` here to avoid warnings about possible multiple enumeration.

Reply:
👍